storage/iscsi-tools/open-iscsi/pkg.yaml

```diff
@@ -33,23 +34,27 @@ steps:
         patch -p1 < /pkg/patches/add-missing-headers.patch
         patch -p1 < /pkg/patches/dont-use-lib64.patch
         patch -p1 < /pkg/patches/remove-werror.patch
+
+        pip3 install ninja
     build:
       - |
         export PKG_CONFIG_PATH=/usr/lib/pkgconfig
 
-        # musl doesn't have glob_onlydir but by spec it's also not guaranteed to return only dirs anyway
+        mkdir -p output
+
         LDFLAGS="$LDFLAGS -L/usr/local/lib" \
```

Comment:
the `make` way of building was already deprecated for `iscsi-tools` for a while, bringing in gcc 13 broke the build, so switch to meson